packages/inspector/src/utils/workflow/dsl/deserialize-dsl-workflow.ts

```
@@ -565,27 +789,31 @@ function nodeToCode(
+      }
 
       case 'filter':
         lines.push(
-          `${indent}const ${flowNode.outputVar} = ${flowNode.sourceVar}.filter((${flowNode.itemVar}) => ${conditionToCode(flowNode.condition)})`
+          `${indent}${bindingPrefix(flowNode.outputVar)}${flowNode.sourceVar}.filter((${flowNode.itemVar}) => ${conditionToCode(flowNode.condition)})`
         )
         lines.push('')
         break
 
       case 'arrayPredicate':
         const method = flowNode.mode === 'some' ? 'some' : 'every'
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)